### DISCUSSION TITLE
Custom Scenarios not showing up!

### DISCUSSION THREAD

**Discussion by tylere661** [OpenRCT2/OpenRCT2#21929] [Q&A]
I have a lot of custom scenarios put in my openrct2 and original RCT2 scenario folders and they appeared in the original game, but not in openrct2.  What is the problem?

**Comment by SpartanFrederic104** [comment]
Did you try deleting the scenarios.idx file in your OpenRCT2 root folder?

**Reply by tylere661** [reply]
No, but I will try that.  Thanks!

**Reply by tylere661** [reply]
I deleted the idx file and nothing happened.  The scenarios appear in the base RCT2 game, but not in openrct2.

**Reply by tylere661** [reply]
I figured it out!  I seen the folder where the scenarios are supposed to go at the very bottom of the scenario list window.  Thanks for your help!
